test/sagemaker_tests/pytorch/inference/conftest.py

```diff
@@ -105,7 +105,7 @@ def pytest_addoption(parser):
     parser.addoption('--docker-base-name', default='pytorch')
     parser.addoption('--region', default='us-west-2')
     parser.addoption('--framework-version', default='')
-    parser.addoption('--py-version', choices=['2', '3'], default=str(sys.version_info.major))
+    parser.addoption('--py-version', choices=['2', '3', '37', '38'], default=str(sys.version_info.major))
```

Comment:
Is it possible for us to remove the dependency on minor python versions going forward? It's tough to scale if we're finding out about these types of issues late

Reply:
^ not merge blocker

Reply:
Agree. Let's make a PR, later, to remove this unnecessary constraint.